Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -12,19 +12,43 @@ class DefaultLocationService: LocationService {
 
     init(mapboxConfiguration: MapboxConfiguration,
          historyLocation: [CLLocation]?,
-         logHandler: LogHandler?) {
-        
+         logHandler: LogHandler?,
+         vehicleProfile: VehicleProfile) {
+
         let directions = Directions(credentials: mapboxConfiguration.getCredentials())
-        NavigationSettings.shared.initialize(directions: directions, tileStoreConfiguration: .default)
-        
+
+        NavigationSettings.shared.initialize(directions: directions,
+                                             tileStoreConfiguration: .default,
+                                             navigatorPredictionInterval: 0,
+                                             statusPollingConfig: .init(unconditionalPollingPatience: .greatestFiniteMagnitude ,
```

Comment:
@ikbalkaya Would you please be able to tell me where you got this `statusPollingConfig` line from? It seems to fix #377, which I've written about a bit in 9e935ad1489bba5d5f95291de19889356ef29828. I'd like to perhaps submit it as a separate fix, but would first like to understand this line a bit better.

Reply:
@lawrence-forooghian This was given as an example code usage in this [issue](https://github.com/mapbox-collab/Ably-DH-Collab/issues/8) . If you have access to it, I also asked for explanation for this field. They said

> Predictions will be disabled with navigationPredictionInterval = 0. And setting unconditionalPollingPatience to the maximum value will prevent the routing engine from doing any simulation when there is no signal, such as in a tunnel or urban canyons. This is a parity of InternalUtils.setUnconditionalPollingPatience(Long.MAX_VALUE) in the https://github.com/mapbox-collab/Ably-DH-Collab/issues/3#issuecomment-1203127239

Reply:
Resolving this as this is a separate conversation I had with Mapbox team in https://github.com/mapbox-collab/Ably-DH-Collab/issues/8